### DISCUSSION TITLE
采用export_model导出SDMGR模型时报错

### DISCUSSION THREAD

**Discussion by yan0102** [PaddlePaddle/PaddleOCR#13010] [Q&A]
我用自己的数据集训练了一个SDMGR模型，预测不报错，
当我使用export_model.py导出推理模型时，报如下错误：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/42789365/e80dd8a9-7ced-4c2d-b787-e503e2726e82)
这是什么原因啊

**Comment by GreatV** [comment]
建议给 [paddle](https://github.com/PaddlePaddle/Paddle) 提一个issue。

**Comment by hoobnn** [comment]
遇到了同样的问题，还不知道怎么解决
